src/storage/src/hummock/validator.rs

```diff
@@ -61,6 +61,7 @@ pub async fn validate_ssts(task: ValidationTask, sstable_store: SstableStoreRef)
             Arc::new(SstableIteratorReadOptions {
                 cache_policy: CachePolicy::NotFill,
                 must_iterated_end_user_key: None,
+                max_preload_retry_times: 0,
```

Comment:
What is `max_preload_retry_times` for?

Reply:
to retry create a new streaming connection when meet network error.